keadm/cmd/keadm/app/cmd/edge/batchjoin.go

```diff
@@ -0,0 +1,179 @@
+package edge
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+	"gopkg.in/yaml.v3"
+)
+
+type Node struct {
+	IP              string `yaml:"ip"`
```

Comment:
The YAML format is disordered. It contains uppercase and lowercase letters, hyphens (-), and underscores (_). We should use the camel case format.